src/lib/y2partitioner/actions/controllers/filesystem.rb

```diff
@@ -373,6 +374,33 @@ def configure_snapper
           filesystem.configure_snapper
         end
 
+        # Paths that are mounted in the current device graph, excluding
+        # subvolumes
+        #
+        # @return [Array<String>]
+        def mounted_paths
+          devices = mounted_devices.reject { |d| d.is?(:btrfs_subvolume) }
+          devices.map(&:mount_path)
+        end
+
+        # Sorted list of the default mount paths to offer to the user
```

Comment:
Are they actually sorted?

Reply:
Yes, following the same criteria than the old partitioner (which contained a really complicated logic to guarantee this same sorting).